### DISCUSSION TITLE
%EB%AA%A8%EB%93%A0-%ED%9A%8C%EC%82%AC%EC%97%90-%EC%96%B4%ED%95%84-%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EC%A7%80%EC%9B%90-%EB%8F%99%EA%B8%B0%EC%99%80-%EB%AA%A9%ED%91%9C%EC%84%A4%EC%A0%95-%EB%B0%A9%EB%B2%95/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#3] [Comments]
# %EB%AA%A8%EB%93%A0-%ED%9A%8C%EC%82%AC%EC%97%90-%EC%96%B4%ED%95%84-%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EC%A7%80%EC%9B%90-%EB%8F%99%EA%B8%B0%EC%99%80-%EB%AA%A9%ED%91%9C%EC%84%A4%EC%A0%95-%EB%B0%A9%EB%B2%95/

Theo's Blog

https://dataportal.kr/%EB%AA%A8%EB%93%A0-%ED%9A%8C%EC%82%AC%EC%97%90-%EC%96%B4%ED%95%84-%ED%95%A0-%EC%88%98-%EC%9E%88%EB%8A%94-%EC%A7%80%EC%9B%90-%EB%8F%99%EA%B8%B0%EC%99%80-%EB%AA%A9%ED%91%9C%EC%84%A4%EC%A0%95-%EB%B0%A9%EB%B2%95/

**Comment by ACACIANh** [comment]
좋은 글 감사합니다

**Comment by binary-ho** [comment]
좋은 글 감사합니다. 꼭 고민해봐야겠네요

**Comment by devFancy** [comment]
좋은 글 감사합니다 ! 👏🏻

**Comment by JaydenLee1116** [comment]
짧지만 임팩트 있는 내용이네요!! 좋은 글 감사합니다 :)!

**Comment by ella-yschoi** [comment]
프론트엔드 개발자 채용공고에서 링크되어있는 글 보고 왔습니다. 덕분에 더 좋은 이력서를 작성할 수 있을 것 같아요. 감사합니다!

**Comment by kimgh06** [comment]
감사합니다. 님이 저를 살렸습니다.
가장 기본적인 것을 갖추지 못하고 그저 제 이익에 눈이 멀어 남의 시선을 보지 못했습니다.

**Comment by joonfluence** [comment]
정말 공감 가면서도 좋은 글이네요 감사합니다 태양님~!

**Comment by hckang80** [comment]
좋은 내용 감사합니다.
즉시 활용하겠습니다.

**Comment by jj5u** [comment]
열심히 고민해 봐야 하는 문제네요. 좋은 글 감사드립니다!
